packages/website/docusaurus.config.js

```diff
@@ -1,168 +1,167 @@
+    ({
+      algolia: {
+        appId: 'QPBQ67WNIG',
+        apiKey: 'b590ae1153bf574215ca1605c19eb1fe',
+        indexName: 'docsearch',
+        contextualSearch: true,
+      },
+      navbar: {
+        hideOnScroll: true,
+        logo: {
+          alt: 'DocSearch',
+          src: 'img/docsearch-logo.svg',
+          srcDark: 'img/docsearch-logo-white.svg',
+        },
+        items: [
+          // left
+          {
+            label: 'Docs',
+            to: 'docs/what-is-docsearch',
+            position: 'left',
+          },
+          // right
+          {
+            type: 'docsVersionDropdown',
+            position: 'right',
+          },
+          {
+            href: 'https://github.com/algolia/docsearch',
+            position: 'right',
+            className: 'header-github-link',
+          },
+        ],
+      },
+      colorMode: {
+        defaultMode: 'light',
+        disableSwitch: false,
+        respectPrefersColorScheme: true,
+      },
+      announcementBar: {
+        id: 'supportus',
+        content:
+          '⭐️ If you like DocSearch, give it a star on <a target="_blank" rel="noopener noreferrer" href="https://github.com/algolia/docsearch">GitHub</a>! ⭐️',
+      },
+      footer: {
+        links: [
+          {
+            title: 'Docs',
+            items: [
+              {
+                label: 'Getting Started',
+                to: 'docs/what-is-docsearch',
+              },
+              {
+                label: 'FAQ',
+                to: 'docs/faq',
+              },
+              {
+                label: 'DocSearch v3',
+                to: 'docs/DocSearch-v3',
+              },
+            ],
+          },
+          {
+            title: 'DocSearch',
+            items: [
+              {
+                label: 'Apply',
+                to: 'apply',
+              },
+              {
+                label: 'Issues',
+                to: 'https://github.com/algolia/docsearch/issues',
+              },
+              {
+                label: 'Privacy',
+                to: 'https://www.algolia.com/policies/privacy/',
+              },
+            ],
+          },
+          {
+            title: 'Community',
+            items: [
+              {
+                label: 'Forum',
+                to: 'https://discourse.algolia.com/tags/docsearch',
+              },
+              {
+                label: 'Discord',
+                to: 'https://discord.com/invite/AW2H22QyJc',
+              },
+            ],
+          },
+          {
+            title: 'Social',
+            items: [
+              {
+                label: 'GitHub',
+                to: 'https://github.com/algolia/docsearch',
+              },
+              {
+                label: 'Twitter',
+                to: 'https://twitter.com/docsearch_',
+              },
+              {
+                label: 'Algolia Blog',
+                to: 'https://blog.algolia.com/',
```

Comment:
blog is now at algolia.com 
we might want to advertise tech blog too

Reply:
```suggestion
                to: 'https://algolia.com/blog/',
```